tasks/rvm.yml

```diff
@@ -53,3 +53,16 @@
   when: not rvm_binary.stat.exists
   become: yes
   become_user: '{{ rvm1_user }}'
+
+- name: Change rvm user/db
+  lineinfile: "dest={{ rvm1_install_path }}/user/db state=present regexp='^{{ item }}=' line='{{ item }}={{ rvm1_configurations[item] }}'"
+  with_items: rvm1_configurations.keys()
```

Comment:
Looks here we are missing conditional like `when: rvm1_configurations.keys() is undefined` or  `when: rvm1_configurations.keys()`